### ISSUE TITLE
Gap between tavern walls and inner corner

### ISSUE THREAD

**Issue by l0calh05t** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#258]
**Describe the bug**
Taver inner corners are too narrow and leave gaps to neighboring tavern walls

**To Reproduce**
Place some tavern walls that require a corner piece, and a corner piece. observe gaps on both sides of corner piece.

**Screenshots/Videos/Twitch Clips**
![image](https://user-images.githubusercontent.com/1749069/79072954-11b9d300-7ce4-11ea-8f73-4a340d56db04.png)

**Comment by LuPro** [comment]
Those inner corners are meant for filling (as the name suggests) inside corners like this: 
![image](https://user-images.githubusercontent.com/20774976/79074211-60b73680-7ceb-11ea-89d5-18b9a436c080.png)

What you are probably searching for is this: 
![image](https://user-images.githubusercontent.com/20774976/79074245-9b20d380-7ceb-11ea-9989-351f2aaa02c9.png) the "Tavern no Floor 1x1 A" (there also exists a version with floor)

**Comment by l0calh05t** [comment]
Not really, there are just some wall-size×wall-size gaps that can't be filled with a regular corner piece. Besides, the gap is there in your first picture as well (although not quite as bothersome as the walls behind them have little/no gap.
